caracal/schema/selfcal_schema.yml

```diff
@@ -143,7 +148,7 @@ mapping:
         example: 'True'
       img_specfit_nrcoeff:
         type: int
-        desc: Number of spectral polynomial terms to fit to each clean component. This is equal to the order of the polynomial plus 1.
+        desc: Number of spectral polynomial terms to fit to each clean component. This is equal to the order of the polynomial plus 1. Use -1 to disable fitting.
```

Comment:
@o-smirnov , could you please add a line here saying that with  `img_specfit_nrcoeff` set to -1, Crystalball won't be able to work?

Reply:
Please see my comment on the -1 option not actually being necessary. In. the code -1 does the same as 0 (0 terms fitted). So I think the desc in the schema should replace -1 with 0.